infrastructure/src/main/java/org/corfudb/infrastructure/logreplication/transport/client/IClientChannelAdapter.java

```diff
@@ -30,9 +30,6 @@ public abstract class IClientChannelAdapter {
     @Getter
     private final LogReplicationClientRouter router;
 
-    @Getter
-    private IChannelContext channelContext;
-
```

Comment:
The import could be removed as well

Reply:
+1

Reply:
+1, also we can remove it from IServerChannelAdapter and completely get rid of this interface like Chris suggested.